en/content_how_to_get_compensated_for_work.md

```diff
@@ -1,47 +1,59 @@
 # How to Get Compensated for Work
 
-## Where do I Find Work?
+The [Smart Contract Research Forum](smartcontractresearch.org) (SCRF) has created “How to Get Compensated for Work” for individuals to learn about SCRF’s payment system.
```

Comment:
Vertical quotes, please.

Also, not sure what the utility is of a sentence that says that scrf has created the doc that people are reading. Everything is explained in the next paragraph.